### DISCUSSION TITLE
noUnusedExports

### DISCUSSION THREAD

**Discussion by hornta** [biomejs/biome#2371] [Ideas]
In ESLint I relied heavily on [no-unused-modules](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unused-modules.md#importno-unused-modules) and it would be nice to have this in Biome as well, as a sibling rule to `noUnusedImports`. I found a [closed discussion](https://github.com/biomejs/biome/discussions/827) of this rule so I'm opening a new.

**Comment by arendjr** [comment]
I agree that would be nice, but it would require whole-project analysis to implement this. That’s on the roadmap for this year, but it will take a bit for it to be done.

**Comment by arendjr** [comment]
There's also a new discussion around this, which is getting a bit more traction, so I'm closing this one: https://github.com/biomejs/biome/discussions/6379
